src/cassandra/mongoose_cassandra_worker.erl

```diff
@@ -225,7 +225,8 @@ handle_call({continue, Req}, From, State = #state{}) ->
     NewState = update_req(NewReq, State),
     {noreply, process_request(RequestId, NewState)};
 handle_call(Msg, _From, State) ->
-    ?WARNING_MSG("Unexpected call ~p", [Msg]),
+    ?LOG_WARNING(#{what => mongoose_cassandra_worker_unexpected_call,
```

Comment:
We don't need `mongoose_` prefix in what. And I doubt we need worker part. So it should be `cassandra_unexpected_call` here and below.